<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformSettings</class>
 <widget class="QWidget" name="TransformSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="Translate">
        <property name="title">
         <string>Translate</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="x">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="translateX"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="translateY"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="translateZ"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="y">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="z">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="GlmVisu::RotationInput" name="Rotate"/>
      </item>
      <item>
       <widget class="GlmVisu::AngleInput" name="angleInput"/>
      </item>
      <item>
       <widget class="QGroupBox" name="Scale">
        <property name="title">
         <string>Scale</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="x_2">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="y_2">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="z_2">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="scaleX"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="scaleY"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="scaleZ"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlmVisu::RotationInput</class>
   <extends>QGroupBox</extends>
   <header>RotationInput.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GlmVisu::AngleInput</class>
   <extends>QGroupBox</extends>
   <header>AngleInput.h</header>
   <slots>
    <slot>changeAngleValue(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>translateX</sender>
   <signal>textEdited(QString)</signal>
   <receiver>TransformSettings</receiver>
   <slot>updatedValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>matrixChanged(glm::mat4)</signal>
  <slot>updatedValue()</slot>
 </slots>
</ui>
